lib/beaker/cli.rb

```diff
@@ -56,14 +58,14 @@ def initialize
     end
 
     #Provision, validate and configure all hosts as defined in the hosts file
-    def provision
+    def provision(validate=true, configure=true)
       begin
         @hosts =  []
         @network_manager = Beaker::NetworkManager.new(@options, @logger)
         @hosts = @network_manager.provision
         @network_manager.proxy_package_manager
-        @network_manager.validate
-        @network_manager.configure
+        @network_manager.validate if validate
+        @network_manager.configure if configure
```

Comment:
It was mentioned on the ticket that we would at some point like to provide users with the ability to prevent validation/configuration of hosts. I'm not sure if you guys had a different approach to implementing this, but for now I just added some flags to the provision method, which default to true